staging.tfvars

```diff
@@ -58,7 +58,7 @@ hpc_resource_provisioner_scratch_bucket       = ""
 
 entitycore_svc_s3_bucket_name            = "entitycore-data-staging"
 entitycore_svc_s3_bucket_allowed_origins = ["*"]
-entitycore_svc_image_url                 = "public.ecr.aws/openbraininstitute/entitycore:2025.6.5"
+entitycore_svc_image_url                 = "public.ecr.aws/openbraininstitute/entitycore:2025.6.6"
```

Comment:
We can update also prod tag in this PR, and close https://github.com/openbraininstitute/aws-terraform-deployment/pull/376
I'll take care of dropping the prod db (that should be empty at the moment)

cc @jdcourcol